### PR TITLE
Fix some typos and rephrase some sentences in README.md

### PR DESCRIPTION
**Pull request by iamthesenate1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,7 +118,6 @@ And its working directory:
 Common problems
 ---------------
 
-Before starting the build process, make sure you have a "Temp" folder in your development directory. If it doesn't exist, create it.
```
Comment (iamthesenate1) [inline README.md:121]: Deleted as it felt redundant.  The Temp folder is included with the repo.

File: README.md [hunk 2]
```diff
@@ -53,9 +53,9 @@ cd Serious-Engine
 
 If you have access to a copy of the game (either by CD or through Steam),
 you can copy the *.gro files from the game directory to the repository.
-Not that for running SS:TFE, you don't need the ModEXT.txt file, it's only for SS:TSE.
+Note that for running SS:TFE, you don't need the ModEXT.txt file, it's only for SS:TSE.
 
-Alternativelly, you can simply copy the whole content of the Windows install as base dir, add the `SE1_10.gro` from the repo in the root folder of the install, and then copy the binary built in `Bin` folder
+Alternatively, you can simply copy the whole content of the Windows install as base dir, add the `SE1_10. gro` from the repo in the root folder of the install, and then copy the binary built in `Bin` folder
```
Comment (ptitSeb) [inline README.md:58]: Why adding a space in the namefile? it's `SE1_10.gro` no space after the dot, it's not a sentense.

Comment (iamthesenate1) [reply README.md]: Oops! Sorry for hurrying, I fixed it now.

**Comment by ptitSeb** [comment]
Thanks, it's merged now.

**Comment by iamthesenate1** [comment]
Thanks for the quick review and merging!

**Pull request merged**